python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -39,14 +39,16 @@ class NCMECCheckpoint(
 
     # The biggest value of "to", and the next "from"
     get_entries_max_ts: int
+    next_fetch: t.Optional[str] = ""
+    last_fetch_time: t.Optional[int] = 0
```

Comment:
blocking: Add a comment on the relationship between the paging URL for why we save this.

ignorable: We could also make this variable explicitly mention the paging_url and not set it if we are not paging